params/config_test.go

```diff
@@ -26,3 +26,24 @@ func TestChainConfig_CheckConfigForkOrder(t *testing.T) {
 	assert.Nil(t, BaobabChainConfig.CheckConfigForkOrder())
 	assert.Nil(t, CypressChainConfig.CheckConfigForkOrder())
 }
+
+func TestChainConfig_Copy(t *testing.T) {
+	a := CypressChainConfig
+	b := a.Copy()
+
+	b.UnitPrice = 0x1111
```

Comment:
Can you add hardfork block number case, and mintingAmount case too?

Reply:
Added in https://github.com/klaytn/klaytn/pull/1428/commits/080445b2ce37c3c65a49e57dcef72c7f6b82c773.